tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -1,138 +1,126 @@
 
 **Directories** allow you to organize and manage your subaccounts according to your technical and business needs. A directory can contain one or more subaccounts. It can't contain other directories. Using directories to group subaccounts is optional - you can still create subaccounts directly under your global account.
 
 **Subaccounts** let you structure a global account according to your project's requirements with regards to members, authorizations, and quotas. Subaccounts in a global account are independent from each other. Each subaccount is associated with a **region**, which is the physical location where applications, data, or services are hosted. The region assigned to your subaccount doesn't have to be directly related to your location. You could be located in the United States, for example, but operate your subaccount in Europe.
 
-In your trial global account, a Cloud Foundry subaccount with an org and a space called dev is automatically created for you.
+In your trial global account, a Cloud Foundry subaccount with an org and a space called `dev` is automatically created for you.
 
-!![Account Structure](AccountStructure.png)
+<!-- border -->![Account Structure](AccountStructure.png)
 
 If you want to get a deeper understanding of the account structure, have a look at the documentation on the [Account Model](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/8ed4a705efa0431b910056c0acdbf377.html).
 
-[VALIDATE_2]
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step: ](Navigate in the cockpit)]
+### Navigate in the cockpit
 You can access your trial account cockpit at [https://cockpit.hanatrial.ondemand.com/trial/](https://cockpit.hanatrial.ondemand.com/trial/).
 
 
-There are 2 ways of navigating in the cockpit:
+There are two ways of navigating in the cockpit:
 
-- using the left hand-side navigation
-- using the breadcrumbs at the top of the page
+- using the left hand-side navigation,
+- or using the breadcrumbs at the top of the page.
 
-There are certain actions you can perform at each hierarchical level (global account, subaccount, space etc.) - we will refer to the sum of these actions as a **scope**. For example, when you are in a global account you can add global account members, create subaccounts and add entitlements to subaccounts. They represent the **global account scope**.
+There are certain actions you can perform at each hierarchical level (global account, subaccount, space etc.). We will refer to the sum of these actions as a **scope**. For example, when you are in a global account, you can add global account members, create subaccounts, and add entitlements to subaccounts. They represent the **global account scope**.
```

Comment:
Don't use "we" unless you are referring to SAP.

Reply:
fixed